### PR TITLE
PLT-3028 Added CTRL+/ to show shortcuts

### PR DESCRIPTION
**Pull request by DavidLu1997**
CTRL+/ now shows the shortcuts command. Using it does not affect what the user has typed into the textbox.

`executeCommand` was moved out of `create_post.jsx` to `actions/channel_actions.jsx` to reduce direct calls on Client.

### CODE REVIEW

**Comment by hmhealey** [comment]
+1

File: webapp/components/create_post.jsx [hunk 1]
```diff
@@ -320,6 +323,24 @@ class CreatePost extends React.Component {
     componentWillUnmount() {
         ChannelStore.removeChangeListener(this.onChange);
         PreferenceStore.removeChangeListener(this.onPreferenceChange);
+        document.removeEventListener('keydown', this.showShortcuts);
+    }
+    showShortcuts(e) {
+        if ((e.ctrlKey || e.metaKey) && e.keyCode === Constants.KeyCodes.FORWARD_SLASH) {
```
Comment (enahum) [inline webapp/components/create_post.jsx]: Will this be executed whenever I press the keyboard shortcuts even if I'm already typing something?

File: webapp/components/create_post.jsx [hunk 2]
```diff
@@ -320,6 +323,24 @@ class CreatePost extends React.Component {
...
+            ChannelActions.executeCommand(
+                this.state.channelId,
+                '/shortcuts ',
+                false,
+                null,
+                (err) => {
+                    const state = {};
+                    state.serverError = err.message;
+                    state.submitting = false;
+                    this.setState(state);
```
Comment (crspeller) [inline webapp/components/create_post.jsx]: Just so you know. You can be more concise here with the literal syntax:
```javascript
this.setState({
    serverError: err.message,
    submitting: false
});
```

File: webapp/components/create_post.jsx (hunk 1 above)
Comment (DavidLu1997) [inline webapp/components/create_post.jsx]: Yes, but it does not affect what you are typing, you do not lose anything by pressing this.

File: webapp/components/create_post.jsx (hunk 2 above)
Comment (DavidLu1997) [inline webapp/components/create_post.jsx]: Changing :+1:

**Comment by enahum** [comment]
+1

**Pull request merged**